luatest/hooks.lua

```diff
@@ -129,6 +130,7 @@ local function run_group_hooks(runner, group, hooks_type)
 end
 
 local function run_test_hooks(self, test, hooks_type, legacy_name)
+    log.info('Run %s hook for %s', hooks_type, test.name)
```

Comment:
1. I would convert it to `verbose`.
2. I propose to use the following format:

`` Run hook `%s` ``

Reply:
I've fixed that.